File: app/src/debug/java/com/zrq/test/point/demo/fragment/MyTestListFragment.java
package com.zrq.test.point.demo.fragment;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.zrq.test.point.annotation.TestEntryPoint;
import com.zrq.test.point.annotation.TestEntryPointListFragment;
import com.zrq.test.point.list.TestListFragment;

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2020/12/23 10:06
 */
@TestEntryPointListFragment
public class MyTestListFragment extends TestListFragment {

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        adapter.addItem("手动添加方法", new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(getContext(), "手动添加方法", Toast.LENGTH_SHORT).show();
            }
        });
    }

    @TestEntryPoint("App-非静态方法")
    public void test1() {
        Toast.makeText(getContext(), "App-非静态方法", Toast.LENGTH_SHORT).show();
    }
}

File: module-kotlin/src/debug/java/com/zrq/test/point/demo/kotlin/fragment/MyTestListFragment.kt
package com.zrq.test.point.demo.kotlin.fragment

import android.widget.Toast
import com.zrq.test.point.annotation.TestEntryPoint
import com.zrq.test.point.annotation.TestEntryPointListFragment
import com.zrq.test.point.list.TestListFragment

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2020/12/23 10:06
 */
@TestEntryPointListFragment
class MyTestListFragment : TestListFragment() {
    override fun onAddTestItems() {
        addItem("module-kotlin-手动添加方法") {
            Toast.makeText(context, "module-kotlin-手动添加方法", Toast.LENGTH_SHORT).show()
        }
    }

    @TestEntryPoint("module-kotlin-非静态方法")
    fun test1() {
        Toast.makeText(context, "module-kotlin-非静态方法", Toast.LENGTH_SHORT).show()
    }
}